src/com/facebook/buck/cxx/CxxLinkableEnhancer.java

```diff
@@ -163,6 +163,14 @@ public static CxxLink createCxxLinkableBuildRule(
     // the soname.
     if (linkType == Linker.LinkType.SHARED) {
       argsBuilder.add(new StringArg("-shared"));
+
```

Comment:
Yes this makes all shared libraries relocatable without using LD_LIBRARY_PATH. Although I think android ignores rpaths anyway.